migrations/env.py

```diff
@@ -0,0 +1,96 @@
+from __future__ import with_statement
```

Comment:
Woah, that is something ancient. Shouldn't be needed.

Reply:
I didn't write it manually, when I run 
**$ python migrate.py db init**
**$ python migrate.py db migrate**
**$ python migrate.py db upgrade**
**Migrations** folder with all it's internal content was created.

Reply:
Understood:)